EchoTestFutures.py

```diff
@@ -84,7 +84,7 @@ def run_echo_test(port_type=SERIAL_TYPE, port_no=SERIAL_PORT,
 
 
 if __name__ == "__main__":
-    # Notice that because this is a benchmark, we have set logging to of the lowest verbose levels
+    # Notice that because this is a benchmark, we have set logging to the lowest verbose level
```

Comment:
This comment is inaccurate, INFO isn't the "lowest verbose" level. Could say "set logging to the INFO level" instead?